### PR TITLE
Sprint2. 1365-01 - Upon clicking the empty checkbox, the point count increases.   

### PR DESCRIPTION
**Pull request by MariaMoroz**
- TC1365-01 - created test and methods in POM (HomePage).   
- Class ProfilePage has been made public

### CODE REVIEW

File: src/test/java/pages/HomePage.java [hunk 1]
```diff
@@ -20,4 +27,45 @@ public TestListPage clickTestsMenu() {
...
+        return this;
+    }
+
+    public HomePage clickWeek1FirstCheckbox() {
+        week1FirstCheckbox.click();
+
+        return this;
+    }
+
+    public HomePage verifyWeek1FirstCheckboxUnchecked() {
```
Comment (ViktoriyaED) [inline src/test/java/pages/HomePage.java:54]: What do you think about naming this method 'uncheckWeek1FirstCheckboxIfChecked' or 'ensureWeek1CheckboxUnchecked'?

File: src/test/java/pages/HomePage.java [hunk 2]
```diff
@@ -4,9 +4,16 @@
 import com.microsoft.playwright.Page;
 import com.microsoft.playwright.Playwright;
 
+
```
Comment (ViktoriyaED) [inline src/test/java/pages/HomePage.java:7]: This empty line may be unnecessary here

File: src/test/java/pages/HomePage.java (hunk 1 above)
Comment (MariaMoroz) [reply src/test/java/pages/HomePage.java:54]: I like this one: ensureWeek1CheckboxUnchecked. Thank you

File: src/test/java/pages/HomePage.java (hunk 2 above)
Comment (MariaMoroz) [reply src/test/java/pages/HomePage.java:7]: removed

**Review by verafes — APPROVED** [review]

**Review by NatalliaOwl — APPROVED** [review]

**Review by ArinaJur — APPROVED** [review]
LGTM

**Pull request merged**